Editor/Scripts/Analytics/MeshsyncAnalytics.cs

```diff
@@ -91,11 +157,84 @@ private struct SyncEventData {
                 entitySyncType = entityTypeStr
             };
 
-            logIfWarning(
-                EditorAnalytics.SendEventWithLimit(
-                    MESHSYNC_SYNC_INSTALLEDEVENTNAME,
-                    eventData,
-                    MESHSYNC_SYNC_VERSION));
+            this.syncEventStack.Push(new TimedEvent<SyncEventData>
+            {
+                LogTime = DateTimeOffset.Now,
+                Data = eventData
+            });
+
+            if (!CoalesceEvents) {
```

Comment:
Is this ever false?

Reply:
Right, if it's never false, then we should simply remove the unused code.